<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="Word_Clock">
       <attribute name="title">
        <string>Word Clock</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>iran</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="iran_show_time_lb">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>140</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>200</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>germany</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="germany_show_time_lb">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>200</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>260</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>usa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="usa_show_time_lb">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>260</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Alarm">
       <attribute name="title">
        <string>Alarm</string>
       </attribute>
       <widget class="QLineEdit" name="input_hour_text_alarm">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>50</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>50</y>
          <width>16</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_minute_text_alarm">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>50</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="add_alarm_button">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>120</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>add</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>521</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="grid_layout_alarm"/>
       </widget>
       <widget class="QLineEdit" name="input_title_linetext">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>11</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Stop_Watch">
       <attribute name="title">
        <string>Stop Watch</string>
       </attribute>
       <widget class="QPushButton" name="stop_stopwatch_button">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>370</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <fontweight>ExtraLight</fontweight>
         </font>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
       <widget class="QPushButton" name="start_stopwatch_button">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <fontweight>ExtraLight</fontweight>
         </font>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
       <widget class="QLabel" name="show_stopwatch_label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>311</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>00 : 00 : 00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Timer">
       <attribute name="title">
        <string>Timer</string>
       </attribute>
       <widget class="QLineEdit" name="input_hour_text">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>170</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>170</y>
          <width>16</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>170</y>
          <width>16</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_minute_text">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>170</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_second_text">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>170</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DSEG7 Modern</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="start_timer_button">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>270</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reset_input_timer_button">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>270</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
